app-backend/src/main/kotlin/org/example/appbackend/service/FeedbackService.kt

```diff
@@ -0,0 +1,10 @@
+package org.example.appbackend.service
+
+import org.example.appbackend.dto.FeedbackDto
+import org.example.appbackend.entity.FeedbackEntity
+
+interface FeedbackService {
+    fun saveFeedback(feedbackDto: FeedbackDto): FeedbackEntity
```

Comment:
It'll be better if saveFeedback returns FeedbackDto (but with extra fields). Because Repository layer works with Entity. Service layer works with Dto

Reply:
You can make this function signature: "fun saveFeedback(feedbackDto: CreateFeedbackDto): FeedbackDto"

Reply:
Agree